docs/manpage.rst

```diff
@@ -289,32 +289,33 @@ Options controlling job submission
 
 .. option:: -A, --account=NAME
 
-   Submit test-related jobs using the account ``NAME``.
-   This option is relevant only for the Slurm backend and translates to Slurm's ``--account`` option and it precedes any options specified in the :js:attr:`access` system partition configuration parameter.
```

Comment:
All these descriptions should remain. You should add the the deprecation node + what this option is equivalent to. For example, for Slurm, this is equivalent to `-J account=NAME`.